ui/lib/pki/addon/components/pki-config/import.hbs

```diff
@@ -0,0 +1,14 @@
+<form {{on "submit" this.submitForm}} data-test-pki-config-import-form>
+  <TextFile @subText="Upload a PEM bundle or certificate" @onChange={{this.onFileUploaded}} @label="Import" />
+  {{yield}}
+  <div class="field is-grouped box is-fullwidth is-bottomless">
+    <div class="control">
+      <button type="submit" class="button is-primary" data-test-pki-config-save>
+        Done
+      </button>
+      <LinkTo @route="overview" class="button has-left-margin-s" data-test-pki-config-cancel>
```

Comment:
If a user clicks "cancel" will this take them back to the overview instead of keep them at `configuration/create` with the three radio buttons? I think we'd want to keep them on the configuration view, regardless of if an issuer exists or not - but I could be misreading this! Perhaps a question for design 🤔